tests/unit/backend/api/data_portal/api_server/base_api_test.py

```diff
@@ -49,52 +45,40 @@ def _remove_timestamps(jrb, removing):
 
 
 class BaseAuthAPITest(BaseAPITest):
-    @staticmethod
-    def get_mock_server_and_auth_config(additional_scope=None, token_duration=0):
-        mock_oauth_server = MockOauthServer(additional_scope, token_duration)
-        mock_oauth_server.start()
-        assert mock_oauth_server.server_okay
-
-        # Use the CorporaAuthConfig used by the app
-        auth_config = CorporaAuthConfig()
-
-        os.environ["API_BASE_URL"] = f"http://localhost:{mock_oauth_server.port}"
-        # Overwrite the environment's auth config with our oidc server's config.
-        authconfig = {
-            "api_base_url": f"http://localhost:{mock_oauth_server.port}",
-            "callback_base_url": auth_config.callback_base_url,
-            "redirect_to_frontend": auth_config.redirect_to_frontend,
-            "client_id": auth_config.client_id,
-            "client_secret": auth_config.client_secret,
-            "audience": auth_config.audience,
-            "api_audience": auth_config.api_audience,
-            "cookie_name": auth_config.cookie_name,
-        }
-        auth_config.set(authconfig)
-        return (mock_oauth_server, auth_config)
```

Comment:
now doing above with simple config instance variable assignment in `setupClass` below